Documentation/Kernel/DevelopmentGuidelines.md

```diff
@@ -164,6 +164,23 @@ To generalize the exception a bit more - debug messages (being used sparingly) w
 assumption of paths could be OK, as long as they never have any functional implication
 on the user.
 
+## Allocation of major numbers for new devices
+
+All major numbers in the operating systems are allocated per device type (family of devices).
+We allocate them in `Kernel/Devices/MajorNumberAllocation.h`, based on this set of rules:
+1. The allocation is either for a new type of block device or character device, not both.
+2. The family name string is not already taken for any block or character devices.
+3. The family name string is informative and short.
+4. The family name string is written in CamelCase name when inserted to the appropriate
+   enum (either CharacterDeviceFamily or BlockDeviceNumber).
+4. The family name string is written in snake_case name when inserted to the appropriate
```

Comment:
You have `4.` twice.

Reply:
That's a bit embarrassing of me :sweat_smile:
I will fix this when sending a new revision.